packages/@aws-cdk/aws-apigateway/README.md

```diff
@@ -868,6 +868,19 @@ By performing this association, we can invoke the API gateway using the followin
 https://{rest-api-id}-{vpce-id}.execute-api.{region}.amazonaws.com/{stage}
 ```
 
+### Importing VPC Link
```

Comment:
We should change this to a section called 'VPC Link' and briefly describe with a code snippet about VPC links.

Sorry about the digression since it's not directly related to your change. Are you able to help out? 
Don't forget to add an entry in the 'table of contents' at the top of the file.

A single sentence at the end of that that existing VPC links can be imported using the `fromVpcLinkId()` API should suffice.

Reply:
@nija-at Yes, of course. I add section about 'VPC Link' and move this in the  'VPC Link' section.